modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -851,9 +854,41 @@ protected function render() {
 		}
 	}
 
+	public function print_child( $index, $item_id = null ) {
+		$children = $this->get_children();
+
+		if ( ! empty( $children[ $index ] ) ) {
+			// Add data-tab-index attribute to the content area.
+			$add_attribute_to_container = function ( $should_render, $container ) use ( $item_id ) {
+					$this->add_attributes_to_container( $container, $item_id );
+
+				return $should_render;
+			};
+
+			add_filter( 'elementor/frontend/container/should_render', $add_attribute_to_container, 10, 3 );
+			$children[ $index ]->print_element();
+			remove_filter( 'elementor/frontend/container/should_render', $add_attribute_to_container );
+		}
+	}
+
+	protected function add_attributes_to_container( $container, $item_id ) {
+		$container->add_render_attribute( '_wrapper', [
+			'role' => 'region',
```

Comment:
We will need to add these attributes inside the editor as well.
See how we have done it inside the nested tabs: look for js/editor/view.js (or similar).

Reply:
Have you found js/editor/view.js?